docs/general/configuration_options.md

```diff
@@ -63,6 +63,10 @@ Possible values:
 - `:singular`
 - `:plural` (default)
 
+##### jsonapi_namespace_separator
+
+Sets separator string for namespaced models to render `type` attribute. Default value is `--`.
+
```

Comment:
An alternative name or perhaps alias would be `dashed_namespace_separator` (which might also make a nice PR into Rails, to make it easier to configure the replacement string from `/` to `--`)